apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/web/controllers/block.ex

```diff
@@ -18,7 +18,8 @@ defmodule OMG.ChildChainRPC.Web.Controller.Block do
   """
 
   use OMG.ChildChainRPC.Web, :controller
-  plug(OMG.ChildChainRPC.Plugs.Health)
+  # check for health before calling action
+  # plug(OMG.ChildChainRPC.Plugs.Health)
```

Comment:
Was you intention to leave it commented? Consider delete lines or uncomment.

Reply:
it used to be enabled, so I commented it out so that we could have a discussion on what to do, cc @pdobacz

Reply:
AFAIK, eth conn isn't required for the chch's `block.get` and `transaction.submit` (at least not to successfully process the request per se, as we should always know and react if eth conn is lost!).

Judging by your comment, you're intending to go with these checks removed and I think let's do it (and I guess let's remove the commented code x2)

Reply:
I would disagree. The service is not functioning properly at the fundamental level - we don't have ethereum connectivity - and we have no idea for how long. Do we really want to introduce state changes when we don't know how long the outage is?

Reply:
Not sure what you're disagreeing with. I didn't really write anything decisive, if you want to block the requests then uncomment, if you want to let them through - remove.

If we prefer to err on the safe side and block even for transient losses of eth-conn, let's uncomment.

If we're confident that via alerts we'll learn that eth-conn's lost and react before it becomes an issue (or stop the services if need be), let's remove.

Probably taking either path would warrant opening an issue to follow-up.